charts/aad-pod-identity/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for pod-aad-identity-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  enabled: true
+
+azureIdentity:
+  name: "azure-identity"
+  type: 0 # type 0: MSI, type 1: Service Principal
+  resourceID: "" # /subscriptions/subscription-id/resourcegroups/resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity-name
+  clientID: "" # 53ba6f2b-6d52-4f5c-8ae0-7adc20808854
```

Comment:
Could you please write something like <client id form the az identity command> instead of id number here...

Reply:
Sure. This is just an example client ID but I understand it's just a GUID so isn't super useful. I'll remove it. Thanks.